<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CertificateDetails</class>
 <widget class="QDialog" name="CertificateDetails">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>570</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
font-family: Roboto, Helvetica;
font-size: 14px;
}
#CertificateDetails {
background-color: #FFFFFF;
border-radius: 4px;
}
#leftPane {
border-right: 1px solid #E7EAEF;
}
#lblCertHeader, #lblCertDetails {
color: #003168;
font-size: 22px;
}</string>
  </property>
  <layout class="QGridLayout" name="CertificateDetailsLayout" columnstretch="0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="leftPaneLayout">
     <property name="spacing">
      <number>24</number>
     </property>
     <property name="leftMargin">
      <number>24</number>
     </property>
     <property name="topMargin">
      <number>32</number>
     </property>
     <property name="rightMargin">
      <number>24</number>
     </property>
     <item>
      <widget class="QLabel" name="lblCertHeader">
       <property name="text">
        <string>Certificate information</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="lblCertInfo">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="pageLayout">
     <property name="spacing">
      <number>24</number>
     </property>
     <property name="leftMargin">
      <number>24</number>
     </property>
     <property name="topMargin">
      <number>32</number>
     </property>
     <property name="rightMargin">
      <number>24</number>
     </property>
     <item>
      <widget class="QLabel" name="lblCertDetails">
       <property name="text">
        <string>Certificate details</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tblDetails">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>320</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QHeaderView::section {
border: 1px solid #E7EAEF;
border-width: 0px 0px 1px 0px;
padding: 0px 0px 8px 5px;
background-color: #FFFFFF;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>26</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>140</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>20</number>
       </attribute>
       <column>
        <property name="text">
         <string>Field</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="detailedValue">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="navigationArea" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#navigationArea {
border-top: 1px solid #E7EAEF;
background-color: #F3F5F7;
}
QPushButton {
padding: 10px 12px;
border: 1px solid #2F70B6;
border-radius: 4px;
color: #ffffff;
background-color: #2F70B6;
font-size: 12px;
font-weight: 700;
}
QPushButton:hover {
border-color: #2B66A6;
background-color: #2B66A6;
}
QPushButton:pressed {
border-color: #215081;
background-color: #215081;
}
QPushButton:disabled {
border-color: #82A9D3;
background-color: #82A9D3;
}
QPushButton:default {
background-color: transparent;
border-color: #AD2A45;
color: #AD2A45;
}
QPushButton:default:hover {
background-color: #F5EBED;
}
QPushButton:default:pressed {
background-color: #E1C1C6;
}</string>
     </property>
     <layout class="QHBoxLayout" name="navigationAreaLayout">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="navigationAreaSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="close">
        <property name="text">
         <string>Close</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
